.gitignore

```diff
@@ -9,6 +9,9 @@ secrets.py
 airflow.db
 
 # Airflow temporary artifacts
+airflow-core/src/airflow/git_version
+airflow-core/src/airflow/ui/coverage/
```

Comment:
Nit, very much optional:

These could be done in the local gitignore `airflow-core/.gitignore` if we wanted?

Reply:
Yeah. I plan a series of smaller cleanups as nextsteps . Added to the list.